### PR TITLE
Fix Capella genesis testing tool

### PR DESCRIPTION
**Pull request by hwwhww**
- Fix `create_genesis_state`: should have updated `previous_version` and `current_version` 
- Add Capella version `initialize_beacon_state_from_eth1`

### CODE REVIEW

**Review by etan-status — APPROVED** [review]

**Comment by etan-status** [comment]
Thanks, this indeed fixes the problems with `make test` in #2938 (merged this PR into there as well)

File: specs/capella/beacon-chain.md [hunk 1]
```diff
@@ -489,3 +490,59 @@ def process_bls_to_execution_change(state: BeaconState,
         + address_change.to_execution_address
     )
 ```
+
+## Testing
+
+*Note*: The function `initialize_beacon_state_from_eth1` is modified for pure Capella testing only.
+Modifications include:
+1. Use `CAPELLA_FORK_VERSION` as the current fork version.
```
Comment (ralexstokes) [inline specs/capella/beacon-chain.md:498]:
```suggestion
1. Use `CAPELLA_FORK_VERSION` as the previous fork version.
```

should this say `previous` as the modification?

File: specs/capella/beacon-chain.md [hunk 2]
```diff
@@ -489,3 +490,59 @@ def process_bls_to_execution_change(state: BeaconState,
...
+    # Set genesis validators root for domain separation and chain versioning
+    state.genesis_validators_root = hash_tree_root(state.validators)
+
+    # Fill in sync committees
+    # Note: A duplicate committee is assigned for the current and next committee at genesis
+    state.current_sync_committee = get_next_sync_committee(state)
+    state.next_sync_committee = get_next_sync_committee(state)
+
+    # Initialize the execution payload header
+    # If empty, will initialize a chain that has not yet gone through the Merge transition
```
Comment (ralexstokes) [inline specs/capella/beacon-chain.md:544]: this is an interesting point.... is there any value in testing a capella net pre-merge?

I think its an open question but given that we are starting this beacon chain at cappella, it feels like testing a pre-merge state is something that would never hit in a public-facing testnet or mainnet

**Review by ralexstokes — COMMENTED** [review]
generally looks good!

left some feedback

File: specs/capella/beacon-chain.md (hunk 1 above)
Comment (hwwhww) [reply specs/capella/beacon-chain.md:498]: Ah, I think it changes both. nice catch!

File: specs/capella/beacon-chain.md (hunk 2 above)
Comment (hwwhww) [reply specs/capella/beacon-chain.md:544]: agreed. I'd remove this comment.

**Review by djrtwo — APPROVED** [review]

**Pull request merged**